go_dependencies.bzl

```diff
@@ -4,9 +4,8 @@ def bb_storage_go_dependencies():
     go_repository(
```

Comment:
This function was named `bb_storage_go_dependencies()`, so that we can distinguish it from all of the other `bb_*_go_dependencies()` functions we have. Now that that's going away due to our use of go.mod, I think we can rename this to plain `go_dependencies()` or something.

Reply:
Done